File: Code Chef/JavaScript/800-899/803 Hardest Problem Bet HARDBET.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const whoWinsBet = (prblm1AnsCount, prblm2AnsCount, prblm3AnsCount) => {
    const hardestPrblm = Math.min(prblm1AnsCount, prblm2AnsCount, prblm3AnsCount);
    
    const winnerOrDraw = (hardestPrblm === prblm1AnsCount)
        ? 'Draw'
        : (hardestPrblm === prblm2AnsCount)
            ? 'Bob' : 'Alice';
    console.log(winnerOrDraw);
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [prblm1AnsCount, prblm2AnsCount, prblm3AnsCount] = testcase;
        whoWinsBet(prblm1AnsCount, prblm2AnsCount, prblm3AnsCount);
    });
});

File: Code Chef/C Programming/800-899/803 Hardest Problem Bet HARDBET.c
#include <stdio.h>
int main(void)
{
	int t,a,b,c;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d %d %d",&a,&b,&c);
	    if((a<b)&&(a<c))
	    {
	        printf("Draw\n");
	    }
	    else if((b<a)&&(b<c))
	    {
	        printf("Bob\n");
	    }
	    else if((c<a)&&(c<b))
	    {
	        printf("Alice\n");
	    }
	}
	return 0;
}
